lambda_src/drivers/process_sentry_logger.py

```diff
@@ -77,16 +77,17 @@ def process_row(
     """Each row is sent to Sentry via the SENTRY_DRIVER_LOGGER
 
     Args:
-        error (str): The actual error message.
-        query_id (str): The query ID of the failed query.
+        account (str): The Snowflake account name.
+        region (str): Region of the Snowflake account.
         database (str): The DB.
         schema (str): The schema.
         name (str): The object name.
-        ts (str): The timestamp of the error.
         history_type (str): pipe or task or query.
-        region (str): Region of the Snowflake account.
-        account (str): The Snowflake account name.
+        error (str): The actual error message.
+        ts (str): The timestamp of the error.
     """
+    CONSOLE_LOGGER.info(f'sentry_logger driver invoked.')
+
     history_url = get_snowsight_url(
```

Comment:
I don't think we should have Snowsight hard-coded into the Sentry driver -- can we move this code to the Snowflake UDF?

Reply:
Yeah that checks out. Makes more sense to be on the Snowflake side.